website/content/docs/connect/gateways/index.mdx

```diff
@@ -37,8 +37,9 @@ Mesh gateways enable the following scenarios:
 
 -> **1.8.0+:** This feature is available in Consul versions 1.8.0 and newer.
 
-Ingress gateways are an entrypoint for outside traffic. They enable potentially unauthenticated ingress traffic from
-services outside the Consul service mesh to services inside the service mesh.
+Ingress gateways enable connectivity within your organizational network from services outside the Consul service mesh
+to services within the mesh. To accept ingress traffic from the public internet, use Consul's
+[API Gateway](https://www.hashicorp.com/blog/announcing-hashicorp-consul-api-gateway) instead.
```

Comment:
**Question for reviewer**: Should I even mention the API Gateway at this point? Or leave it out for now?

Reply:
I think it's good to start getting people aware of Consul API Gateway. I vote we keep it.